articles/managed-operations/man-ref-customer-prereqs.md

```diff
@@ -0,0 +1,132 @@
+
+- Internet Control Message Protocol (ICMP)
+
+#### Linux & Windows (Agentless)
+
+- Simple Network Management Protocol (SNMP) - Port 161 over UDP
+
+- Internet Control Message Protocol (ICMP)
+
+### Operating system services
+
+SNMP is required for agentless monitoring.
+
+> [!NOTE]
+>
+> The SNMP service **must** be configured to start on boot.
+
+#### Linux
+
+On Linux, you can install SNMP from the operating system's public repository.
+
+#### Windows
+
+On Windows, you can install SNMP through the Roles and Features wizard.
+
+### VPN configuration
+
+Managed Monitoring as a Service relies on an IPsec VPN being configured on your environment's edge appliance.
+
+UKCloud will provide you with the VPN standards, including the pre-shared key, peer endpoint and peer subnet to configure the VPN.
+
+## Anti-Virus as a Service
+
+Anti-Virus as a Service provides you with the confidence that your environments are protected from the dangers of viruses and malware, all without the usual headaches associated with the deployment, management, and operation of an antivirus solution. For more information see the [*Anti-Virus as a Service Service Scope*](man-sco-antivirus.md).
+
+UKCloud will provide you with the installation *package(s)* and commands to install and configure the antivirus agent on your virtual machines.
+
+## Patching as a Service
+
+Patching as a Service provides routine OS patch management for compute instances. It helps you accelerate business value by removing the burden of basic OS hygiene, letting you focus your attention above the operating system. For more information, see the [*Patching as a Service Service Scope*](man-sco-patching.md).
+
+### Operating systems
+
+UKCloud only supports in-life, vendor-backed operating systems, such as Microsoft Windows, Red Hat Enterprise Linux and Canonical Ubuntu. Customer-built operating systems (for example Gentoo Linux custom build) are unsupported by this service.
+
+### Customer environment configuration
+
+#### Linux
+
+To provide secure communication between UKCloud's centralised patch management platform and your virtual machine, we use key-based SSH sessions, reducing the need to share credentials. As such, you'll need to fulfil the following prerequisites on each virtual machine subscribing to this service:
+
+- An allow rule for port 22 (SSH) on your virtual machine's firewall (for example, iptables).
+
+- A local user account with passwordless sudo access. You'll need to share details of this account with UKCloud when onboarding to this service.
+
+- The `/etc/ssh/sshd_config` configuration file must be configured to allow SSH access for the patching user you created in the step above.
+
+- UKCloud will provide you with a public SSH key during the onboarding process. You'll need to append this key to the `authorized_keys` file, located at `~/.ssh/authorized_keys` within the home directory of the user you created in the step above.
```

Comment:
"As part of the onboarding process, UKCloud is willing to perform this action, as long as the above requirements are already met."